### DISCUSSION TITLE
Video embed plugin?

### DISCUSSION THREAD

**Discussion by laserpilot** [honkit/honkit#371] [Q&A]
I am trying to migrate my gitbook https://blair-neal.gitbook.io/survey-of-alternative-displays/ to honkit but I use a **lot** of the gitbook Embed tag throughout (GitHub is here https://github.com/laserpilot/Survey_of_Alternative_Displays )

Does anyone have a recommended plugin for supporting YouTube/vimeo video embeds within honkit? I'd like to make minimal formatting changes if possible, but I can't find an obvious answer. This plugin seems close but I can't tell if it would embed videos properly yet https://www.npmjs.com/package/gitbook-plugin-url-embed 

Thank you!

**Comment by azu** [comment]
HonKit support GitBook plugins.
If you install it, it will probably work as is.

Install:

```
npm install gitbook-plugin-url-embed --save-dev
```

Edit `book.json`

```
{
    "plugins": ["url-embed"]
}
```

**Reply by laserpilot** [reply]
Thank you! With such a long original document, is there an easy way to test without a failed build or serve message? Or should I just take part of it out and test that way?
